lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystem.cs

```diff
@@ -154,6 +154,7 @@ public IFileSystem GetAzureFileSystem()
                 TestCloudBlobClient.BaseUri.ReturnsForAnyArgs(FakeUri);
                 TestCloudBlockBlob.Open().Returns(TestOpenStream);
                 TestCloudBlockBlob.Create().Returns(TestCreateStream);
+                TestCloudBlockBlob.Blob.ReturnsForAnyArgs(new CloudBlockBlob(new Uri("https://stg.blob.core.windows.net/container/file")));
```

Comment:
nit: FakeUri as well as "container/file" are used throughout this test file. If it's not too much trouble, I would use statics instead of magic strings.

Reply:
removed